docker/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM registry.tudelft.nl/samxl/humble_dev_base:0.0.2 as pose_estimation_builder
+
+RUN sudo apt-get update && \
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)